kong.conf.default

```diff
@@ -741,6 +741,37 @@
                                  # Detailed discussion of this behavior is
                                  # available in the online documentation.
 
+# When using Postgres as the backend storage, you can optionally enable Kong
```

Comment:
These comments don't match the style of the rest of the document — I'm not pointing this out for stylistic reasons though: I'm afraid they won't be formatted correctly by the generator script that generates [this page](https://docs.konghq.com/2.0.x/configuration/#properties-reference) based on this file.

If you want to add an intro prose section to the documentation, this should be added to [this file instead](https://github.com/Kong/docs.konghq.com/blob/master/autodoc-conf/data.lua) — (yes, we want to eventually move this file and the script to the main Kong repo like we did for `autodoc-admin-api`), and the comments in this file should be formatted as descriptions for each entry so they generate the "Properties reference" in the documentation website.

Reply:
@hishamhm Thanks for the note, I did not realize the importance of formatting in here first.

I have moved this paragraph up to the "DATASTORE" section instead of the `data.lua` file. It seems to makes more sense to me as it clearly falls under such category, plus we also explain things such as DB-less mode in there so it should follows the current pattern after the move. 

I also changed the comment style of the properties to make sure autodoc understands them. Here is a preview of the generated Markdown after the change: https://gist.github.com/dndx/807132b43edd58f478a9d9deafe4d30a. Seems that everything is rendering correctly.